assets/benchmarks/shaders/FoveationBenchmarkFragSizeEXT.frag

```diff
@@ -0,0 +1,72 @@
+
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#version 450
+#extension GL_EXT_fragment_invocation_density : enable
+
+layout(location = 0) out vec4 outColor;
+
+layout(binding = 0) uniform Params
```

Comment:
It seems like the benchmark uses both hlsl and glsl shaders? I think having glsl samples is great. Although in this case it is a little confusing. One is using hlsl and and include file, and this glsl shader seems to also duplicate code in FoveationBenchmark.hlsli. Would it be cleaner if we just use hlsl and include common code?

Reply:
I believe this has to do with the way the extensions work for shaders.
The sample uses two different methods to compare, each of them uses his own extension.
I *belive* that `GL_EXT_fragment_invocation_density` was only know in GLSL, hat is why `FoveationBenchmarkFragSizeEXT.frag` was required.
Now, since in HSLS the custom is to use a single file for shader stage, to construct that pipeline the `FoveationBenchmarkFragSizeEXT.vert` was required.
Having said that Benson will have the best answer/opinion about this.

Reply:
I have just ask him via chat and he confrimed my theory.

Reply:
maybe come comments in the glsl shader file to explain why we are using glsl instead of hlsl so that people won't be confused in the future. Also have you checked with Steven regarding the plan to add this in hlsl?

Reply:
Sorry, was there a comment added? I don't see it

Reply:
I just add it in the new commit.